packages/react/src/components/CopyButton/CopyButton.js

```diff
@@ -6,8 +6,9 @@
  */
 
 import PropTypes from 'prop-types';
-import React, { useState, useRef, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import classnames from 'classnames';
+import debounce from 'lodash/debounce';
```

Comment:
I believe we use the direct package in our `dependencies` instead of the namespace

```suggestion
import debounce from 'lodash.debounce';
```